elastic/kibana
# fixed field data sourcing

## Summary

This fixes:
SDH issue **882**
https://github.com/elastic/kibana/issues/173627

### Before the fix
Note how the field actually displays an alias instead of the value. This is due some fallback behavior I think.
![highlighted_before](https://github.com/elastic/kibana/assets/11671118/c66ef18f-d0f9-4964-b69f-94bba6d31c60)

### After the fix
![highlighted_after](https://github.com/elastic/kibana/assets/11671118/ee730392-7617-4654-87bc-a8addb7342e2)


The missing bit was the actual value to display as it was not passed down correctly from the parent conext

### How to test this?

1. create custom index and populate it with data

```
PUT test

POST test/_doc
{
  "user.id": "888",
  "@timestamp": "2024-02-21T15:20:10.084Z"
}
```

2. create threshold rule looking like this:

![image](https://github.com/elastic/kibana/assets/11671118/07089ba8-e0b9-40f1-8372-15cb8a94e043)

3. actual value that triggered the alert should be rendered in the highlighted fields panel in the  flyout
